webapp/src/Entity/Visualization.php

```diff
@@ -0,0 +1,86 @@
+<?php declare(strict_types=1);
+namespace App\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * A team output on a specific testcase.
+ */
+#[ORM\Entity]
+#[ORM\Table(options: [
+    'collation' => 'utf8mb4_unicode_ci',
+    'charset' => 'utf8mb4',
+    'comment' => 'Team output visualization.',
+])]
+#[ORM\Index(columns: ['judgingid'], name: 'judgingid')]
+class Visualization
```

Comment:
If we allow a visualizer to fail gracefully (see exit code 43 message above), we need to be able to record this here (and perhaps a message?).